lib/backend/dynamo/README.md

```diff
@@ -13,14 +13,24 @@ It should be covered by the free tier.
 ### Building
 
 DynamoDB backend is not enabled by default. To enable it you have to 
-compile Teleport with `dynamo` build flag.
+compile Teleport with `dynamodb` build flag.
 
 To build Teleport with DynamoDB enabled, run:
 
 ```
-ADDFLAGS='-tags dynamodb' make teleport
+ADDFLAGS='-tags dynamodb' make build/teleport
```

Comment:
I don't think this is true anymore, DynamoDB support seems to be built by default.

Reply:
Similarly for Firestore if you're looking into this.

Reply:
Certainly the tests are not run build by default - but yeah, you're correct re: the backend implementation itself. Will update the `README`.

Is there a reason why the tests are excluded?

Reply:
Historically, I think the tests weren't run because you needed an AWS instance to do it. We have [an open issue](https://github.com/gravitational/teleport/issues/4673) to run these tests using the DynamoDB emulator though, and we recently added the Firestore emulator to the build pipeline (which introduced Java as a dependency), so this is probably a bit closer to getting done.